spec/Body/SignalingLights.vspec

```diff
@@ -0,0 +1,12 @@
+# (C) 2022 BMW Group - All rights reserved.
+# AUTHOR: Gerrit Grotewold BMW Group;
+
+IsOn:
+  datatype: boolean
+  type: actuator
+  description: Indicates if light is on or off. True = On. False = Off.
```

Comment:
In case of singaling lights, should the description rather be something like inidicates wheter light is "signaling" ?